src/components/Plugin/DaoModule/CustomBlock.vue

```diff
@@ -13,26 +13,50 @@
         <PluginDaoModuleTransactionPreview :transaction="tx" />
       </div>
     </div>
-    <UiButton
+    <div
       v-if="questionDetails?.questionId"
-      @click="showQuestion"
-      class="width-full mb-2"
+      class="mb-3 p-4 border rounded-2 text-white text-center"
     >
-      Show Question
-    </UiButton>
+      <div>
+        <Label> {{ approvalData.decision }} </Label>
+      </div>
+      <div v-if="questionState === 3">
+        <Label> {{ approvalData.timeLeft }} </Label>
+      </div>
+      <div>
+        <UiButton
+          v-if="canChangeAnswer"
+          @click="modalApproveDecisionOpen = true"
+          class="mb-2"
+        >
+          Change Answer
+        </UiButton>
+      </div>
+    </div>
     <UiButton
       v-if="moduleAddress"
       @click="performAction"
       :disabled="!actionEnabled"
       v-text="actionLabel"
       class="width-full button--submit"
     />
+    <teleport to="#modal">
+      <ModalOptionApproval
+        :open="modalApproveDecisionOpen"
+        :isApproved="questionDetails?.isApproved"
+        :bond="questionDetails?.currentBond"
+        :questionId="questionDetails?.questionId"
+        @setApproval="voteOnQuestion"
+        @close="modalApproveDecisionOpen = false"
+      />
+    </teleport>
   </Block>
 </template>
 
 <script>
 import Plugin from '@snapshot-labs/snapshot.js/src/plugins/daoModule';
 import { sleep } from '@/helpers/utils';
+import { format } from 'timeago.js';
```

Comment:
I saw that we have an `_ms` function which implement this, but AFAIK it is only accesible from the HTML (inside of `<template>`). Please let me know if I am missing something :-D

Reply:
You can use the _ms function in the script too, try with `this._ms()`.